RAP/rap-statistics.qmd

```diff
@@ -1040,21 +1040,44 @@ You might be thinking that it would be nice if there was software that could hel
 
 **What does this mean?**
 
-A README is a text file (.txt) that introduces and explains a project. It contains information that is required to understand what the project is about and how to use it.
+A README is a markdown file (.md) that introduces and explains a project. It contains information that is required to understand what the project is about and how to use it. Markdown (.md) files are used for READMEs because they support formatting and render nicely on platforms like GitHub and Azure Devops, meaning that users can see them on the main page of the repository. You can find guidance on basic markdown syntax on the [Markdown Guide](https://www.markdownguide.org/basic-syntax/){target="_blank" rel="noopener noreferrer"}. 
 
 **Why do it?**
 
 It's an easy way to answer questions that your audience will likely have regarding how to install and use your project and also how to collaborate with you.
 
 **How to get started**
 
-As a starting point, you should aim to have as many of the following sections as are applicable to your project:
+For new projects, you can use the create_project function in DfER. Set create_publication_proj to TRUE to create a pre-populated project with a customer folder structure, including a README template.  You can find more information on this in the [DfE R reference](https://dfe-analytical-services.github.io/dfeR/reference/create_project.html). 
 
-- Introduction
-- Requirements (access, software, skills/knowledge)
-- How to use
-- How to contribute
-- Contact details
+If you are creating your own README for existing projects, you should aim to have as many of the following sections as are applicable to your project:
```

Comment:
I think this is too friendly 😆 I think we should say "all of the following sections" or something similar